src/plugins/data/public/ui/filter_bar/_global_filter_group.scss

```diff
@@ -55,3 +55,12 @@
     margin-top: $euiSize * -1;
   }
 }
+
+.globalFilterGroup__removeAllFilters {
```

Comment:
Can you please create an issue on OUI to allow setting the `color` in OuiContextItem?

Reply:
https://github.com/opensearch-project/oui/issues/1345